mypyc/ops.py

```diff
@@ -1470,7 +1470,7 @@ def __init__(self, name: str, module_name: str, is_trait: bool = False,
         # base_mro is the chain of concrete (non-trait) ancestors
         self.base_mro = [self]  # type: List[ClassIR]
 
-        # Children of this class
+        # Subclasses of this class
         self.children = []  # type: List[ClassIR]
```

Comment:
Children only includes immediate children right now, which needs to change for this to work